build.gradle

```diff
@@ -1,13 +1,14 @@
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '2.2.5.RELEASE'
-    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+    id 'org.springframework.boot' version '2.2.5.RELEASE'  apply false
 }
 
+targetCompatibility = sourceCompatibility = 11
```

Comment:
sorry, I meant inside of the subprojects section, as you need to address modules with this setting